### PR TITLE
New spec for mobile app - Blog post support

### PR DESCRIPTION
**Pull request by YuriHechter**

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -194,10 +194,102 @@ const FEED_SUGGESTER_JSON_V3 = `
         }
 `
 
+const FEED_SUGGESTER_JSON_V4_WITH_SINGLE_BLOG_POST = `
+{
+	"name": "SortSuggester",
```
Comment (bbfsdev) [inline api/handlers.go:199]: You have two sort suggesters one inside the other. Looks like not intended.

File: api/handlers.go [hunk 2]
```diff
@@ -194,10 +194,102 @@ const FEED_SUGGESTER_JSON_V3 = `
...
+	"specs": [
+		{
+			"name": "SortSuggester",
+			"specs": [
+				{
+					"name": "RoundRobinSuggester",
```
Comment (bbfsdev) [inline api/handlers.go:205]: Round robin suggester for only one child suggester does not make sense

File: api/handlers.go [hunk 3]
```diff
@@ -194,10 +194,102 @@ const FEED_SUGGESTER_JSON_V3 = `
...
+									]
+								}
+							]
+						}
+					]
+				}
+			]
+		}
+	]
+}
```
Comment (bbfsdev) [inline api/handlers.go:285]: I think the following is equivalent:




{
  "name": "SortSuggester",
  "specs": [
    {
      "name": "CompletionSuggester",
      "specs": [
        {
          "name": "LimitSuggester",
          "specs": [
            {
              "name": "DataCollectionsSuggester",
              "filters": [
                {
                  "filter_selector": 1,
                  "args": [
                    "DAILY_LESSON"
                  ]
                }
              ]
            }
          ],
          "args": [
            "1"
          ]
        },
        {
          "name": "SortSuggester",
          "specs": [
            {
              "name": "DataCollectionsSuggester",
              "filters": [
                {
                  "filter_selector": 1,
                  "args": [
                    "DAILY_LESSON"
                  ]
                }
              ]
            },
            {
              "name": "DataContentUnitsSuggester",
              "filters": [
                {
                  "filter_selector": 0,
                  "args": [
                    "VIDEO_PROGRAM_CHAPTER",
                    "MEAL",
                    "FRIENDS_GATHERING",
                    "ARTICLE"
                  ]
                }
              ]
            },
            {
              "name": "LimitSuggester",
              "specs": [
                {
                  "name": "DataContentUnitsSuggester",
                  "filters": [
                    {
                      "filter_selector": 0,
                      "args": [
                        "BLOG_POST"
                      ]
                    }
                  ]
                }
              ],
              "args": [
                "1"
              ]
            }
          ]
        }
      ]
    }
  ]
}

Comment (YuriHechter) [reply api/handlers.go:275]: fixed

**Review by bbfsdev — APPROVED** [review]

**Pull request merged**